jinja2/filters.py

```diff
@@ -940,6 +940,37 @@ def do_attr(environment, obj, name):
     return environment.undefined(obj=obj, name=name)
 
 
+@contextfilter
+def do_applymacro(*args, **kwargs):
+    """Applies a macro by reference instead of directly.
```

Comment:
Use ReStructuredText formatting for docstrings. Render the docs with `pip install pallets-sphinx-themes` then `make html` from the docs folder. You can view the build by opening `_build/html/index.html` in a browser.